sample-templates/deploy-openai-model.json

```diff
@@ -0,0 +1,59 @@
+{
+  "name": "Deploy OpenAI Model",
+  "description": "Deploy a model using a connector to OpenAI",
+  "use_case": "PROVISION",
+  "version": {
+    "template": "1.0.0",
+    "compatibility": [
+      "2.12.0",
+      "3.0.0"
+    ]
+  },
+  "workflows": {
+    "provision": {
+      "nodes": [
+        {
+          "id": "create_openai_connector",
+          "type": "create_connector",
+          "user_inputs": {
+            "name": "OpenAI Chat Connector",
+            "description": "The connector to public OpenAI model service for GPT 3.5",
+            "version": "1",
+            "protocol": "http",
+            "parameters": {
+              "endpoint": "api.openai.com",
```

Comment:
```suggestion
              "endpoint": "api.openai.com",
              "response_filter": "$.choices[0].message.content",
```

Reply:
@joshuali925 just wondering is this a new optional parameter on connector API? I don't see it any of the examples on documentation website

Reply:
see https://github.com/opensearch-project/ml-commons/blob/79b9dfd7eeef051fbf9d0d35242052433b83b297/docs/tutorials/agent_framework/build_your_own_chatbot.md#31-create-agent, it's the same parameter for agent/tools